hummingbot/connector/exchange/ndax/ndax_utils.py

```diff
@@ -9,5 +10,65 @@
 DEFAULT_FEES = [2, 2]
 
 
+# USE_ETHEREUM_WALLET not required because default value is false
+# FEE_TYPE not required because default value is Percentage
+# FEE_TOKEN not required because the fee is not flat
+
+
 def convert_to_exchange_trading_pair(hb_trading_pair: str) -> str:
     return hb_trading_pair.replace("-", "")
+
+
+KEYS = {
```

Comment:
Should we include an optional config for account id?

Reply:
Yes. In fact we need to ask for the name of the account the user wants to use (they could have more than one account associated with their user account)